js/plugins/dotprompt/src/index.ts

```diff
@@ -14,14 +14,53 @@
  * limitations under the License.
  */
 
+import { genkitPlugin, PluginProvider } from '@genkit-ai/core';
+
 import { readFileSync } from 'fs';
 import { basename } from 'path';
-
 import { defineDotprompt, Dotprompt } from './prompt.js';
-import { lookupPrompt } from './registry.js';
+import { loadPromptFolder, lookupPrompt } from './registry.js';
 
 export { defineDotprompt, Dotprompt };
 
+export interface DotpromptPluginOptions {
+  promptDir: string;
```

Comment:
```suggestion
  dir: string;
```

It's the Dotprompt plugin, so the directory is naturally for prompts. Also this should have a default value of `./prompts`.

Reply:
Done